packages/@okta/idx-js/idx.js

```diff
@@ -1,2 +1,8224 @@
+(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
hmm how did you generate this file?. Doesn't look like its the minified version.

Reply:
@nikhilvenkatraman-okta  It's not. This is just for testing purpose. I will need to replace it with the minified version once the idx js PR is approved. I don't know what's the development workflow for that repo btw. How do I get it merged?

Reply:
we merge idx-js PRs directly from github.